### DISCUSSION TITLE
HELP for a total newbie to get a .exe

### DISCUSSION THREAD

**Discussion by Nietopp** [community/community#23380] [Other Feature Feedback, Questions, & Ideas]
<p>Hello everyone.<br>
I’m new here. I wanted to download a cool project about stylometry and finding the authors of anonymous texts, but there is no .exe version and I have no idea about how i can make one.<br>
I downloaded Git but couldn’t make heads or tails about what command to use. Then i downloaded visual studio community, I imported the project, and I’m now lost about how to use it.</p>
<p>The project is here : <a href="https://github.com/evllabs/JGAAP.git" class="inline-onebox" rel="noopener nofollow ugc">GitHub - evllabs/JGAAP: The Java Graphical Authorship Attribution Program</a></p>
<p>I would really appreciate an explaination or a .exe cauz I spent my afternoon on it and I’m about to give up.</p>
<p>Thanks for your patience and understanding since I don’t know code</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>From looking at the repository that appears to be a Java program, so there is no .exe. However, the <a href="https://github.com/evllabs/JGAAP/releases">releases page</a> seems to have have JAR files ready to use.</p>
<p>So the easiest way would probably be to</p>
<ol>
<li>download the latest JAR,</li>
<li>install Java,</li>
<li>run the JAR file using Java.</li>
</ol>

**Comment by Nietopp** [comment]
<p>Thank you so much! I feel a bit dumb :sweat_smile:</p>
<p>It was really nice to take a look to give me a hand :wink:</p>

**Comment by tuves** [comment]
<p>No need to feel that way, <a class="mention" href="/nietopp">@Nietopp</a>!</p>
<p>Please know that we’re <em>all</em> learning something new every day :smiley: and thank goodness for awesome people like <a class="mention" href="/airtower-luna">@airtower-luna</a> who’s willing to give us a hand when we need it. I’m sure you’ll be on the helping end in no time. Welcome to the GitHub Community!</p>
